Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM nartayzhanybekov/tchaikovski:latest
```

Comment:
tchaikovsk**y**?

Reply:
I think it'd be better if you put it into some shared account and share credentials to it